src/core/context.ml

```diff
@@ -24,45 +24,68 @@ let is_function ls t = L.mem ls t.functions
 let get_ls t = get_ls_env t.env
 let get_ts t = get_env ns_find_ts t.env
 
-let stdlib =
+let builtins =
   [
     ([ "None" ], "None");
     ([ "Some" ], "Some");
     ([ "[]" ], "[]");
     ([ "infix ::" ], "(::)");
     ([ "infix =" ], "(=)");
-    ([ "prefix !" ], "!");
-    ([ "Gospelstdlib"; "infix +" ], "Ortac_runtime.Z.add");
-    ([ "Gospelstdlib"; "infix -" ], "Ortac_runtime.Z.sub");
-    ([ "Gospelstdlib"; "infix *" ], "Ortac_runtime.Z.mul");
-    ([ "Gospelstdlib"; "infix /" ], "Ortac_runtime.Z.div");
-    ([ "Gospelstdlib"; "mod" ], "Ortac_runtime.Z.rem");
-    ([ "Gospelstdlib"; "pow" ], "Ortac_runtime.Z.pow");
-    ([ "Gospelstdlib"; "logand" ], "Ortac_runtime.Z.logand");
-    ([ "Gospelstdlib"; "prefix -" ], "Ortac_runtime.Z.neg");
-    ([ "Gospelstdlib"; "infix >" ], "Ortac_runtime.Z.gt");
-    ([ "Gospelstdlib"; "infix >=" ], "Ortac_runtime.Z.geq");
-    ([ "Gospelstdlib"; "infix <" ], "Ortac_runtime.Z.lt");
-    ([ "Gospelstdlib"; "infix <=" ], "Ortac_runtime.Z.leq");
-    ([ "Gospelstdlib"; "integer_of_int" ], "Ortac_runtime.Z.of_int");
-    ([ "Gospelstdlib"; "abs" ], "Ortac_runtime.Z.abs");
-    ([ "Gospelstdlib"; "min" ], "Ortac_runtime.Z.min");
-    ([ "Gospelstdlib"; "max" ], "Ortac_runtime.Z.max");
-    ([ "Gospelstdlib"; "succ" ], "Ortac_runtime.Z.succ");
-    ([ "Gospelstdlib"; "pred" ], "Ortac_runtime.Z.pred");
-    ([ "Gospelstdlib"; "Array"; "make" ], "Ortac_runtime.Array.make");
-    ([ "Gospelstdlib"; "Array"; "length" ], "Ortac_runtime.Array.length");
-    ([ "Gospelstdlib"; "Array"; "get" ], "Ortac_runtime.Array.get");
-    ([ "Gospelstdlib"; "Array"; "for_all" ], "Ortac_runtime.Array.for_all");
   ]
 
+(** Map a name from the Gospel parser to an OCaml name when possible
+
+    The strategy is as follows, always dropping the "*fix " prefix of the
+    operator name:
+    - infix operators are kept as is
+    - prefix operators are prefixed with ~
+    - mixfix operators are dropped
+    - other names are kept as is
+
+    TODO? maybe, some filtering should be performed also on other names, such as
```

Comment:
Yes, I think we should / will have to add some filtering.

For now, we send the user a warning when the specifications contain a call to a function that is not implemented in the ortac_runtime (we know about this at translation time because the association list is kept in sync with the ortac_runtime implementation). With these modifications, we will loose these warnings and the programs using the instrumented code may not work (a possibility we should keep avoiding :-)).

One way to go would be to have the default ortac_runtime provide the maximal translatable subset of the gospel stdlib. Then, 
1. we can know at translation time when not to translate the clause and send a warning instead (the current behaviour, but with more functions),
2.  as other runtimes can't add functions that we are not aware of when translating the terms.